querydsl-r2dbc/src/main/java/com/querydsl/r2dbc/JDBCTypeMapping.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright 2015, The Querydsl Team (http://www.querydsl.com/team)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.querydsl.r2dbc;
+
+import com.google.common.collect.ImmutableSet;
+import com.mysema.commons.lang.Pair;
+import com.querydsl.sql.types.Null;
+
+import javax.annotation.Nullable;
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.sql.Blob;
+import java.sql.Time;
+import java.sql.Timestamp;
+import java.sql.Types;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * {@code JDBCTypeMapping} defines a mapping from JDBC types to Java classes.
+ *
+ * @author tiwe
+ */
+final class JDBCTypeMapping {
```

Comment:
Just use the one from `querydsl-dsl` . Make it `public` if you have to.

Reply:
see next, here we found that Blob/Clob has a special type for r2dbc - im 99% sure this will change when we change/map to correct r2dbc types (for example r2dbc heavily depends on java.time api)